src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/MinerAndroid.java

```diff
@@ -155,6 +146,7 @@ private void breakBlock(BlockMenu menu, Collection<ItemStack> drops, Block block
             }
         } else {
             block.setType(Material.AIR);
+            BlockStorage.clearBlockInfo(block);
         }
```

Comment:
Look a bit closer at the control flow here.
This will not run if the optimization toggle was enabled.